wgsl/index.bs

```diff
@@ -10541,196 +10541,452 @@ See [[#function-calls]].
 ## Logical Built-in Functions ## {#logical-builtin-functions}
 
 <table class='data'>
-  <thead>
-    <tr><th>Parameterization<td style="width:40%;">Overload<td>Description
-  </thead>
-  <tr algorithm="vector all">
+  <tr>
+    <th scope="row" style="width:10%">Parameterization
```

Comment:
I feel like it would be better to swap the description and parameterization rows.